jarviscli/plugins/text_analysis.py

```diff
@@ -0,0 +1,18 @@
+import numpy
+import nltk
+from plugin import plugin
+from nltk import word_tokenize, sent_tokenize
+nltk.download('punkt')
```

Comment:
Should be downloaded in setup.sh. Compare line 74 setup.sh.

Reply:
Could you, please, move the dependencies to the setup.sh for me?? i´m not sure how to manage this task